### PR TITLE
[WIP] Dark/light theme buttons

### PR DESCRIPTION
**Pull request by Mooshua**
Added ability to switch from dark mode to bright mode and vice versa. Thought this might be useful. 
**Since I have no data on bright mode or what pcall returns, I have pcall-ed and tostring-ed most of it. Sorry.**

### CODE REVIEW

File: workshop.lua [hunk 1]
```diff
@@ -18,9 +18,30 @@ local menuEdit = menuBarTop:createItem("Edit")
...
 local menuInsertBlock = menuInsert:createItem("Block")
 
+darkTheme:mouseLeftPressed(function ()
+	-- Set theme to dark mode when dark mode button clicked.
+	workshop.interface:setTheme(enum.themes.dark)
+end)
+brightTheme:mouseLeftPressed(function ()
+	-- Set theme to bright/blind mode when bright mode button clicked.
+	local success,message = pcall(function () 
+		workshop.interface:setTheme(enum.themes.light)
```
Comment (Mooshua) [inline workshop.lua]: *dont know if this is the actual code for it, FYI*

**Comment by teverse** [comment]
Also no need to pcall

**Comment by Mooshua** [comment]
So the Enum..Light was correct? good.

**Comment by Mooshua** [comment]
Also thanks for deprecating light theme, it'll save a lot of clueless devs.

**Pull request closed without merging**